portfolio.html

```diff
@@ -12,41 +12,84 @@
 	 	<nav class="navholder">
 			<div class="logo"><a href="#"><img class="logoimg" alt="" src=></a></div>
 			<div class="home"><a href="#">HOME</a></div>
-			<div class="about"><a href="#">ABOUT ME</a></div>
-			<div class="portfolio"><a href="#">PORTFOLIO</a></div>
-			<div class="contact"><a href="#">CONTACT ME</a></div>
+			<div class="about"><a href="#aboutme">ABOUT ME</a></div>
+			<div class="portfolio"><a href="#portfolio">PORTFOLIO</a></div>
+			<div class="contact"><a href="#contactme">CONTACT ME</a></div>
 		</nav>
 		</nav>
 		<div class="introbox">
 			<div>
-			<h2>HELLO!!!</h2>
+			<h2>Hello There</h2>
 			<h3>Welcome to my world!</h3>
-			<p>A short description of my world to expand on the assigned topic and eventually close the paragraph tag... eventually </p>
+			<p>A short description of my world </p>
 		</div>
 		</div>
-		<h2>ABOUT ME</h2>
+		<h2 id="aboutme">ABOUT ME</h2>
 		<div class="aboutmecontainer">
-			<div class="imgcontainer"><img class ="abmimg "src="/Users/academy/Documents/Assets/panda.jpg" alt="about me img" height="240" width="160"></div>
+			<div class="imgcontainer"><img class ="abmimg "src="/Users/academy/Documents/portfolio/Assets/panda.jpg" alt="about me img" height="240" width="160"></div>
 				<div class="contentcontainer">
-				<h3>placeholder</h3>
+				<h3>Placeholder</h3>
 				<p>It’s time to reveal the face of the magician behind the curtains. Your personal bio should include a nice picture of yours with a short description. Be completely honest about who you are, why you are doing this and how do you plan on providing value to your customers, clients, or prospects.</p>
 				</div>
 			</div>
-			<h2>Portfolio</h2>
-		<div class="aboutmecontainer">
-			<div class="imgcontainer"><img class ="abmimg "src="/Users/academy/Documents/Assets/panda.jpg" alt="about me img" height="240" width="160"></div>
-				<div class="contentcontainer">
-				<h3>placeholder</h3>
-				<p>It’s time to reveal the face of the magician behind the curtains. Your personal bio should include a nice picture of yours with a short description. Be completely honest about who you are, why you are doing this and how do you plan on providing value to your customers, clients, or prospects.</p>
+			<h2 id="portfolio">Portfolio</h2>
+		<div class="portcontainer">
+			<div class="rowcontainer">
+				<div class="columncontainer">
+					<div class="imgcontainer"><img class ="abmimg "src="/Users/academy/Documents/portfolio/Assets/lights.jpeg" alt="about me img" height="160" width="280">
+					</div>
+						<div class="portcontainer">
+							<h3 class="porttitle">Placeholder</h3>
+								<p>It’s time to reveal the face of the magician behind the curtains. </p>
+						</div>
+				</div>
+				<div class="columncontainer">
+					<div class="imgcontainer"><img class ="abmimg "src="/Users/academy/Documents/portfolio/Assets/lights.jpeg" alt="about me img" height="160" width="280">
+					</div>
```

Comment:
```suggestion
					<div class="imgcontainer">
					             <img class ="abmimg "src="/Users/academy/Documents/portfolio/Assets/lights.jpeg" alt="about me img" height="160" width="280">
					</div>
```
indentation possibly?